content/en/agent/guide/linux-agent-2022-key-rotation.md

```diff
@@ -30,7 +30,7 @@ Your host automatically trusts the new key (no further action is required) if yo
 - Containerized Agents (Docker/Kubernetes) for any version
 - Windows/MacOS Agents for any version
 
-Additionally, if you are installing the DEB Agent v7.31.0+ package through `apt` from the `apt.datadoghq.com` repository, your hosts should have the `datadog-signing-keys` package installed, which automatically adds the new key. If you have `datadog-signing-keys` version 1.1.0 or later installed, no further action is needed. If you are relying on the `datadog-signing-keys` package, upgrading to [version 1.1.0](#datadog-signing-keys-version-110) is required to ensure preparedness of your hosts for the key rotation.
+Additionally, if you are installing the DEB Agent v6.31.0+ or v7.31.0+ package through `apt` from the `apt.datadoghq.com` repository, your hosts should have the `datadog-signing-keys` package installed, which automatically adds the new key. If you have `datadog-signing-keys` version 1.1.0 or later installed, no further action is needed. If you are relying on the `datadog-signing-keys` package, upgrading to [version 1.1.0](#datadog-signing-keys-version-110) is required to ensure preparedness of your hosts for the key rotation. Installing DEB Agent v6.35.1+ or v7.35.1+ through `apt` from the `apt.datadoghq.com` repository ensures that `datadog-signing-keys` 1.1.0 is installed.
```

Comment:
I'm not sure I understand the idea of the table. What would be the rows and columns?

Reply:
I personally don't feel like this would be much better, because we'd have 2 rows, the `datadog-signing-keys` version would only be defined in one cell and everything else would be comments anyway - so the comments column would have a lot of text and the rest of the table would feel completely empty compared to that IMO (unless I'm misunderstanding what you mean).

Reply:
I don't get the table TBH. The action should always be "ensure datadog-signing-keys is >= 1.1.0". One way to do it is to upgrade agent to 6.35.1/7.35.1 as this PR documents. I honestly find this table misleading/confusing more than the paragraph. But I've been dealing with this for months now, so maybe that's just me :)

Reply:
(For example, if someone did a clean install of 6.31.0, they would get the latest version of datadog-signing-keys and they'd be fine)

Reply:
I also think the current paragraph is a bit long and it can be confusing, but on the other hand a table might give this too much relevance when it doesn't have it. Maybe there's some solution in-between, like rewording and shortening the parapgraph?

Reply:
Maybe we don't need to mention the v7.31 detail and just focus on 7.35.1? Something like:

```suggestion
Additionally, the `datadog-signing-keys` package version 1.1.0 or later automatically adds the new key. This package is installed when installing the DEB Agent v6.35.1+ or v7.35.1+ through `apt` from the `apt.datadoghq.com` repository.  If you are relying on the `datadog-signing-keys` package, [version 1.1.0](#datadog-signing-keys-version-110) is required to ensure preparedness of your hosts for the key rotation.
```

Although I don't like the repetition too much, I will let @bkabrda figure that out 😄

Reply:
Ok, I tried to rewrite the paragraph and leave some things out. Let me know if you guys think it's better now.

Reply:
LGTM!